docker-compose.yml

```diff
@@ -26,16 +26,16 @@ services:
   #   volumes:
   #     - ./web:/web
 
-  nginx:
-    container_name: nginx
-    image: nginx:latest
-    ports:
-      - 80:8080
-    volumes:
-      - ./nginx/nginx-proxy.conf:/etc/nginx/conf.d/default.conf:ro
-      - ./web/build:/var/www/frontend # maps frontend build inside nginx
-    depends_on:
-      - django
+  # nginx:
+  #   container_name: nginx
+  #   image: nginx:latest
+  #   ports:
+  #     - 80:8080
+  #   volumes:
+  #     - ./nginx/nginx-proxy.conf:/etc/nginx/conf.d/default.conf:ro
+  #     - ./web/build:/var/www/frontend # maps frontend build inside nginx
+  #   depends_on:
+  #     - django
 
```

Comment:
Is there a reason we are keeping this here when it's being commented out? Are there plans to reintroduce this code snippet?

Reply:
We can remove them for sure!